packages/manager/apps/dedicated/client/app/dedicated/server/monitoring/monitoring.routing.js

```diff
@@ -27,8 +27,20 @@ export default /* @ngInject */ ($stateProvider) => {
             ),
           ),
         ),
-      languageEnum: /* @ngInject */ (models) =>
-        models.data.models['dedicated.server.AlertLanguageEnum'].enum,
+      languageEnum: /* @ngInject */ ($translate, models) =>
+        models.data.models['dedicated.server.AlertLanguageEnum'].enum.sort(
+          (a, b) => {
+            const translateA = $translate.instant(`language_${a}`);
+            const translateB = $translate.instant(`language_${b}`);
+            if (translateA < translateB) {
+              return -1;
+            }
+            if (translateB < translateA) {
+              return 1;
+            }
+            return 0;
```

Comment:
**nitpick:** 
```suggestion
            if (translateA < translateB) {
              return -1;
            } else if (translateB < translateA) {
              return 1;
            } else {
              return 0;
            }
```

Reply:
I have an alert eslint if i apply this:
Unnecessary 'else' after 'return'.eslint[no-else-return](https://eslint.org/docs/rules/no-else-return)